test/data_5bus_pu.jl

```diff
@@ -612,6 +612,23 @@ interruptible(nodes5_nodes) = [
     ),
 ]
 
+shiftable5(nodes5) = [
```

Comment:
@akody can you add this data to the TestData repository please for usage in other repos.